### ISSUE TITLE
Consider a stable / dev branch model

### ISSUE THREAD

**Issue by azukaar** [airsonic-advanced/airsonic-advanced#777]
This isn't a specific bug, but Airsonic-advanced is getting a lot of work done (which is great), and a lot of regressions seems to happen from those daily updates. While this is amazing to have updates, it has also the caveat to make the software unstable for users who uses it as a primary player.

The "stable" tag is not currently working as the docker image is missing Curl, and hasn't been updated since April 2020.
I was wondering what would be the right way to use airsonic-advanced to avoid the uncertainty of the updates, and if it would be relevant for the development to move forward with a more stable branch that would update less often?

Thanks for any infos / recommandations, and keep up the good work!

**Comment by dlehammer** [comment]
I agree with @azukaar, as a newcomer looking for the latest stable [docker release](https://hub.docker.com/r/airsonicadvanced/airsonic-advanced) the current outlined official version tagging strategy is rather confusing when compared to the available release tags :/

![image](https://user-images.githubusercontent.com/17439718/172189351-7aa71cc4-f853-43ee-948e-fcde3ad15d08.png)

As the latest [stable-tag](https://hub.docker.com/r/airsonicadvanced/airsonic-advanced/tags?page=1&name=stable) is currently approximately 2 years old.

![image](https://user-images.githubusercontent.com/17439718/172190535-f844ada5-b921-4c71-80bd-1ea76330d84a.png)


@randomnicode [commented](https://github.com/airsonic-advanced/airsonic-advanced/issues/673#issuecomment-979378116) a related snippet on #673:
> You should use Edge versions. Stable is now incredibly old.

I must admit I'm personally unable to commit to a non-deterministic approach, like a `edge-latest`/SNAPSHOT dynamic tag that is shared across 5 releases for the latest release:
![image](https://user-images.githubusercontent.com/17439718/172193092-7cd93a86-8245-42be-a3cf-5a016655c95a.png)
According to conventions the current tagging approach reflects a work-in-progress (going 2+ years) that's subject to change until a stable version released

Seemingly there's a CI-pipeline in place, so the developers should be able to leverage a release-plugin and ex. semantic-versioning or similar going forward 🙏

**Comment by protist** [comment]
This confusion is reflected in Arch Linux's AUR, where there are currently [two packages](https://aur.archlinux.org/packages?K=airsonic-advanced), one tracking the two-year-old [stable release](https://aur.archlinux.org/packages/airsonic-advanced-bin), and the other one tracking the [un-tagged](https://aur.archlinux.org/packages/airsonic-advanced-git) master branch. Actually, I'm not even sure if the latter is the same as the "latest" (recommended on the Docker page) and the "Edge versions" (recommended in the quote above).

Either way, in general the naming of the stable Arch package is exactly that: the stable version that most people should install. The `-git` version is generally not recommended and prone to breakage.
